plugin/src/test/kotlin/tanvd/grazi/GraziTestBase.kt

```diff
@@ -27,6 +29,13 @@ abstract class GraziTestBase(private val withSpellcheck: Boolean) : LightCodeIns
         while (ApplicationManager.getApplication().messageBus.hasUndeliveredEvents(GraziStateLifecycle.topic)) {
             Thread.sleep(500)
         }
+
+        GraziConfig.update { it.copy(enabledLanguages = it.enabledLanguages + Lang.RUSSIAN) }
```

Comment:
Hm, probably we should do it once before all tests and never remove Russian? Like "add Russian if is not already added"

Reply:
Done